src/utils.ts

```diff
@@ -446,6 +453,21 @@ function isHeadlessEnabled() {
     return headless;
 }
 
+/**
+ * Replaces the workspaceFolder placeholder in a specified path, returns the
+ * given path with file disk path.
+ * @param customPath The path that will be replaced.
+ */
+function replaceWorkSpaceFolderPlaceholder(customPath: string) {
+    if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders[0].uri.toString()) {

```

Comment:
small NIT here:
might be worth assigning `vscode.workspace.workspaceFolders[0]` or `vscode.workspace.workspaceFolders[0].uri.toString()` to a `const` for readability's sake.  This might be confusing since we are specifically using the `0` index of the `workspaceFolders` array without an explanation for what it is.

This can be resolved through a comment as well if you prefer that method.